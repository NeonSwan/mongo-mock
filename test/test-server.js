var mongodb = require("../");

console.log("server:");
console.log("  host:27017 -> " + new mongodb.Server("host", 27017));
console.log("");
