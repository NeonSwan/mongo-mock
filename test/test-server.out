server:
  host:27017 -> host:27017

